docs/setup/settings.asciidoc

```diff
@@ -63,3 +63,67 @@ information and all requests.
 The minimum value is 100.
 `status.allowAnonymous`:: *Default: false* If authentication is enabled, setting this to `true` allows
 unauthenticated users to access the Kibana server status API and status page.
+
```

Comment:
As mentioned earlier all of this could go in `install/docker.asciidoc` serving as one place where all docker related info can be found.